plugins/nginx-vhosts/functions

```diff
@@ -22,6 +22,34 @@ fn-nginx-access-log-path() {
   fn-plugin-property-get-default "nginx" "$APP" "access-log-path" "${NGINX_LOG_ROOT}/${APP}-access.log"
 }
 
+fn-nginx-proxy-buffer-size() {
+  declare desc="get the configured proxy buffer size"
+  declare APP="$1"
+
+  fn-plugin-property-get-default "nginx" "$APP" "proxy-buffer-size" "$(getconf PAGESIZE)"
+}
+
+fn-nginx-proxy-buffering() {
+  declare desc="get the configured proxy buffering"
+  declare APP="$1"
+
+  fn-plugin-property-get-default "nginx" "$APP" "proxy-buffering" "on"
+}
+
+fn-nginx-proxy-buffers() {
+  declare desc="get the configured proxy buffers"
+  declare APP="$1"
+
+  fn-plugin-property-get-default "nginx" "$APP" "nginx-proxy-buffers" "8 $(getconf PAGESIZE)"
```

Comment:
Should the `8` always be there?

Reply:
The default value of [`proxy_buffers`](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffers) is `8 4k|8k`, where the first value indicates the number of buffers and the second indicates the size of each buffer (defaults to one memory page size).

Reply:
Should the `8` be computed?